src/subdomains/supporting/notification/interfaces/index.ts

```diff
@@ -26,6 +26,9 @@ export interface MailRequestInput {
   prefix?: TranslationItem[];
   table?: Record<string, string>;
   suffix?: TranslationItem[];
+  from?: string;
+  displayName?: string;
+  banner?: string;
```

Comment:
Maybe rename `MailRequestInput` to `MailRequestUser` and add a new interface for `MailRequestPersonal`?